selftests/deployment/roles/avocado/tasks/main.yml

```diff
@@ -9,4 +9,5 @@
 
 - name: Avocado examples must be installed via specific method
   include_tasks: "roles/avocado/tasks/{{ method }}/examples.yml"
-  when: method is defined
+  when:
+    - method == 'pip' or method == 'copr'
```

Comment:
All possible values for `method` are covered by the *actual* implementations are `pip` and `rpm`, so I don't think this change is necessary (unless I'm missing something).

Reply:
@clebergnu if you look at the full commit, once I remove the "official" repos, nothing is providing python3-avocado-examples. So this is not a permanent change, I'd like to know if the repos are remove or what repos are meant to be used as "official" to fix this.

Reply:
> @clebergnu if you look at the full commit, once I remove the "official" repos, nothing is providing python3-avocado-examples. So this is not a permanent change, I'd like to know if the repos are removed or what repos are instead meant to be used as "official" to fix this.

What I mean is the "official" should really become the Fedora/EPEL packages.  Then, `python-avocado-examples` should be available.